docs/installation.rst

```diff
@@ -15,9 +15,9 @@ PyPy 5.4+ on these operating systems.
 
 * x86-64 CentOS 7.x
 * x86-64 Fedora (latest)
-* macOS 10.13 High Sierra, 10.14 Mojave
-* x86-64 Ubuntu 14.04, 16.04, and rolling
-* x86-64 Debian Jessie (8.x), Stretch (9.x), Buster (10.x), and Sid (unstable)
+* macOS 10.15 Catalina
+* x86-64 Ubuntu (latest)
```

Comment:
We're definitely testing on xenial still (all the tests that don't use docker in travis)

Reply:
fixed